lib/Pages/Welcome.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:pandasenp_flutter/Pages/Login.dart';
+
+class WelcomePage extends StatelessWidget {
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xFFEDECF2),
+      body: SingleChildScrollView(
+        child: Container(
+          margin: const EdgeInsets.symmetric(
+            horizontal: 30,
+            vertical: 120,
+          ),
+          child: Column(
+            children: [
+              Container(
```

Comment:
Prefer SizedBox if no other features of container than height and/or width is used. Saves some ressources.